gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -228,12 +225,12 @@ protected void processMessage(DecodeableKafkaRecord<byte[],byte[]> message) {
             this.eventProducer.emitObservabilityEvent(jobStatus);
           }
 
-          if (this.dagProcEngineEnabled) {
+          if (this.dagProcEngineEnabled && isJobLevelStatus(jobName)) {
```

Comment:
do u have test for isJobLevelStatus or this monitoring method?

Reply:
added in KafkaAvroJobStatusMonitorTest